django_freeradius/tests/base/test_api.py

```diff
@@ -313,6 +313,61 @@ def test_accounting_list_200(self):
         self.assertEqual(item['nas_ip_address'], '172.16.64.91')
         self.assertEqual(item['calling_station_id'], '5c:7d:c1:72:a7:3b')
 
+    def test_accounting_filter_200(self):
```

Comment:
use `@freeze_time(START_DATE)` as in other tests (look up on google to find out what this library does)